src/main/java/com/github/fishio/achievements/AchievedController.java

```diff
@@ -0,0 +1,101 @@
+
+public class AchievedController {
+	
+	private Achievement enemyKill = new Achievement("enemyKill");
+	private Achievement playerDeath = new Achievement("playerDeath");
+	private Log log = Log.getLogger();
+	private Achieve achievementchecker = new Achieve();
+	
+	/**
+	 * Controls the viewer for the second achievement: enemy kills.
+	 */
+	public void setEnemyKill() {
+
+		achievementchecker.checkEnemyKill(enemyKill);
+		if (enemyKill.getLevel() >= 1) {
+			System.out.println("SetenemyKill lvl1 reached");
+			log.log(LogLevel.INFO, "Survival of the fittest achievement level 1 gained!");
+			((AchievementScreenController) Preloader.loadScreen("achievementScreen").getProperties().get("Controller"))
+					.onEnemyKill(1);
+			((SinglePlayerController) Preloader.loadScreen("singlePlayer").getProperties().get("Controller"))
+					.showAchievePopup();
+		}
+		if (enemyKill.getLevel() >= 2) {
+			((AchievementScreenController) Preloader.loadScreen("achievementScreen").getProperties().get("Controller"))
+					.onEnemyKill(2);
+			((SinglePlayerController) Preloader.loadScreen("singlePlayer").getProperties().get("Controller"))
+					.showAchievePopup();
+			log.log(LogLevel.INFO, "Survival of the fittest achievement level 2 gained!");
+		}
+		if (enemyKill.getLevel() >= 3) {
+			((AchievementScreenController) Preloader.loadScreen("achievementScreen").getProperties().get("Controller"))
+					.onEnemyKill(3);
+			((SinglePlayerController) Preloader.loadScreen("singlePlayer").getProperties().get("Controller"))
+					.showAchievePopup();
+			log.log(LogLevel.INFO, "Survival of the fittest achievement level 3 gained!");
+		}
+		if (enemyKill.getLevel() >= 4) {
+			((AchievementScreenController) Preloader.loadScreen("achievementScreen").getProperties().get("Controller"))
+					.onEnemyKill(4);
+			((SinglePlayerController) Preloader.loadScreen("singlePlayer").getProperties().get("Controller"))
+					.showAchievePopup();
+			log.log(LogLevel.INFO, "Survival of the fittest achievement level 4 gained!");
+		}
+		if (enemyKill.getLevel() >= 5) {
+			((AchievementScreenController) Preloader.loadScreen("achievementScreen").getProperties().get("Controller"))
+					.onEnemyKill(5);
+			((SinglePlayerController) Preloader.loadScreen("singlePlayer").getProperties().get("Controller"))
+					.showAchievePopup();
+			log.log(LogLevel.INFO, "Survival of the fittest achievement level 5 gained!");
+		}
+	}
+	
+	/**
+	 * Controls the viewer for the second achievement: enemy kills.
+	 */
+	public void setPlayerDeath() {
+		achievementchecker.checkPlayerDeath(playerDeath);
+		if (playerDeath.getLevel() >= 1) {
+			log.log(LogLevel.INFO, "Survival of the fittest achievement level 1 gained!");
+			((AchievementScreenController) Preloader.loadScreen("achievementScreen").getProperties().get("Controller"))
+					.onPlayerDeath(1);
+			((SinglePlayerController) Preloader.loadScreen("singlePlayer").getProperties().get("Controller"))
+					.showAchievePopup();
```

Comment:
See my previous comment. You can do this method almost the same way.

<logic, layout